prefetch.js

```diff
@@ -109,16 +107,21 @@
       return true;
     }
 
+    function createLinkTag(url){
+      var link = document.createElement('link');
+      link.setAttribute('rel', 'prefetch');
+      link.setAttribute('href', url);
+      return link;
+    }
+
     function injectPrefetchLink(a){
-      if(a && isPrefetchable(a)){
-        var url = (typeof a === 'object') ? a.href : a;
-        var link = document.createElement('link');
-        link.setAttribute('rel', 'prefetch');
-        link.setAttribute('href', url);
+      var url = (typeof a === 'object') ? a.href : a;
+      var link = (url) ? createLinkTag(url) : null;
+      if(link){
         document.getElementsByTagName('head')[0].appendChild(link);
```

Comment:
Nothing changed here but you could just do `document.head` instead of querying the DOM. I don't know which would be faster but it might be easier to read (nit picky thing though)